pipelines/services/signedpipelines.go

```diff
@@ -0,0 +1,104 @@
+package services
+
+import (
+	"encoding/json"
+	"net/http"
+
+	"github.com/jfrog/jfrog-client-go/auth"
+	"github.com/jfrog/jfrog-client-go/http/jfroghttpclient"
+	"github.com/jfrog/jfrog-client-go/utils/errorutils"
+	"github.com/jfrog/jfrog-client-go/utils/io/httputils"
+	"github.com/jfrog/jfrog-client-go/utils/log"
+)
+
+type SignedPipelinesService struct {
+	client *jfroghttpclient.JfrogHttpClient
+	auth.ServiceDetails
+}
+
+const (
+	Artifact = iota
+	BuildInfo
+	ReleaseBundle
+)
```

Comment:
To create an enum in Go, do the following:
```go
type ArtifactType int

const (
	Artifact ArtifactType = iota
	BuildInfo
	ReleaseBundle
)
```

Then you can use the ArtifactType type instead of "int"